src/topp/FeatureFinderMultiplex.cpp

```diff
@@ -226,6 +227,8 @@ class TOPPFeatureFinderMultiplex :
       defaults.setMinInt("missed_cleavages", 0);
       defaults.setValue("knock_out", "false", "Is it likely that knock-outs are present? (Supported for doublex, triplex and quadruplex experiments only.)", ListUtils::create<String>("advanced"));
       defaults.setValidStrings("knock_out", ListUtils::create<String>("true,false"));
+      defaults.setValue("type_m","peptide","The type of averagine to use, currently RNA, DNA or peptide");
```

Comment:
maybe make this an advanced option (see knock_out parameter).
I would also use a more expressive name than "type_m" maybe "averagine_type"